src/index.js

```diff
@@ -139,8 +149,13 @@ class DropdownTreeSelect extends Component {
     })
   }
 
-  onTagRemove = id => {
-    this.onCheckboxChange(id, false)
+  onTagRemove = (id, isKeyboardEvent) => {
```

Comment:
I don't see any call site that passes `isKeyboardEvent`. What am I missing?

Reply:
https://github.com/dowjones/react-dropdown-tree-select/blob/5edbc88ddb8c39e8966b3b04cb8832a210297cb7/src/tag/index.js#L23

Reply:
If you delete by keyboard we set the next focus, not if clicked by mouse.